bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java

```diff
@@ -117,6 +120,8 @@ public String getName() {
     public static final Unit<RadiationDoseEffective> SIEVERT = addUnit(Units.SIEVERT);
     public static final Unit<CatalyticActivity> KATAL = addUnit(Units.KATAL);
     public static final Unit<Speed> METRE_PER_SECOND = addUnit(Units.METRE_PER_SECOND);
+    public static final Unit<Speed> KNOT = addUnit(new TransformedUnit<>("knot", Units.KILOMETRE_PER_HOUR,
```

Comment:
according to the wikipedia the ISO standard symbol for knot is `kn`. See https://en.wikipedia.org/wiki/Knot_(unit)